cli/util/time.rs

```diff
@@ -14,9 +14,10 @@ pub fn utc_now() -> chrono::DateTime<chrono::Utc> {
   let now = std::time::SystemTime::now()
     .duration_since(std::time::UNIX_EPOCH)
     .expect("system time before Unix epoch");
-  let naive = chrono::NaiveDateTime::from_timestamp(
+  let naive = chrono::NaiveDateTime::from_timestamp_opt(
     now.as_secs() as i64,
     now.subsec_nanos(),
-  );
+  )
+  .unwrap();
```

Comment:
This change was made to silence a deprecation warning caused by bumping the version of `chrono`, but I'm a little worried about the `.unwrap`... Should we just ignore the deprecation or is there a better way of handling this?